QuizAppNew/View/QAview.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  QAview.swift
+//  QuizAppNew
+//
+//  Created by Jannatun Nahar Papia on 20/8/22.
+//
+
+import SwiftUI
+
+//MARK: Question Answer view
+struct QAview: View {
+    //outgoing
+    @Binding var correct: Int
+    @Binding var wrong: Int
+    @Binding var answered: Int
+    var set : String
+    @State var data = Qustion()
+    
+//    @State var answered: Int = 0
+    var body: some View {
+        
+        ZStack {
+            
+            if data.question.isEmpty{
+                ProgressView()
+            }
+            else {
+       // IMPORTANT: Top progress view.....
+        ZStack(alignment: Alignment(horizontal: .leading, vertical: .center),
+               content: {
+            Capsule()
+                .fill(Color.gray.opacity(0.5))
+                .frame(height: 6)
+            
+            Capsule()
+                .fill(Color.green)
+                .frame(width: 100, height: 6)
+        })
+        .padding([.leading,.trailing,.top],10)
+        
+        //MARK: Correct and Wrong count....
+        HStack{
+            Label(
+               title: { Text(correct == 0 ? "" : "\(correct)")
```

Comment:
why set business logic correct == 0 , you are only responsible for making UI desig